### ISSUE TITLE
Parsing of configurationfiles changed in icingaweb2 2.5.0

### ISSUE THREAD

**Issue by bcogel** [Icinga/icingaweb2#3281]
In icingaweb2 2.5.0 the parsing of configuration files has changes and causes errors if statements and comments are mixed. At least in commandtransports.ini. 

## Expected Behavior

When using the API for commandtransport a reschedule of a service or host check should trigger the check.

## Current Behavior

When using the API for command transport, a reschedule of a service or host check fails with a stacktrace. Same happens when sending an acknowledgement or a comment (we're syncing them via API between several servers). We've copied and modified the template from https://www.icinga.com/docs/icingaweb2/latest/modules/monitoring/doc/05-Command-Transports/ and we left the inline comment intact.

...
icinga2: Got invalid JSON response from the Icinga 2 API: Syntax error: '<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>400 Bad Request</title>
</head><body>
<h1>Bad Request</h1>
<p>Your browser sent a request that this server could not understand.<br />
</p>
<hr>
<address>Apache/2.2.15 (Red Hat) Server at XXX Port 443</address>
</body></html>
'.

#0 /usr/share/icingaweb2/modules/monitoring/application/forms/Command/Object/CheckNowCommandForm.php(70): Icinga\Module\Monitoring\Command\Transport\CommandTransport->send(Object(Icinga\Module\Monitoring\Command\Object\ScheduleServiceCheckCommand))
#1 /usr/share/php/Icinga/Web/Form.php(1154): Icinga\Module\Monitoring\Forms\Command\Object\CheckNowCommandForm->onSuccess()
#2 /usr/share/icingaweb2/modules/monitoring/library/Monitoring/Web/Controller/MonitoredObjectController.php(264): Icinga\Web\Form->handleRequest()
#3 /usr/share/icingaweb2/modules/monitoring/library/Monitoring/Web/Controller/MonitoredObjectController.php(59): Icinga\Module\Monitoring\Web\Controller\MonitoredObjectController->setupQuickActionForms()
#4 /usr/share/icingaweb2/modules/monitoring/application/controllers/ServiceController.php(66): Icinga\Module\Monitoring\Web\Controller\MonitoredObjectController->showAction()
#5 /usr/share/icingaweb2/library/vendor/Zend/Controller/Action.php(507): Icinga\Module\Monitoring\Controllers\ServiceController->showAction()
#6 /usr/share/php/Icinga/Web/Controller/Dispatcher.php(76): Zend_Controller_Action->dispatch('showAction')
#7 /usr/share/icingaweb2/library/vendor/Zend/Controller/Front.php(937): Icinga\Web\Controller\Dispatcher->dispatch(Object(Icinga\Web\Request), Object(Icinga\Web\Response))
#8 /usr/share/php/Icinga/Application/Web.php(407): Zend_Controller_Front->dispatch(Object(Icinga\Web\Request), Object(Icinga\Web\Response))
#9 /usr/share/php/Icinga/Application/webrouter.php(104): Icinga\Application\Web->dispatch()
#10 /usr/share/icingaweb2/public/index.php(4): require_once('/usr/share/php/...')
#11 {main}
...

Reason for this: 
icingaweb2 is somehow sending the comment through the API (analysis from a colleague, haven't checked myself).

A line with a solitary comment works fine.

## Possible Solution

Correct the documentation and/or review the parsing of the configuration files 

## Steps to Reproduce (for bugs)

1. modify configuration to use API for commandtransport
2. include a comment in commandtransport.ini in a line with a valid statement like in the documentation
3.
4.

## Your Environment

* Icinga Web 2 version and modules (System - About):
* Version used (`icinga2 --version`):
* Operating System and version:
* Enabled features (`icinga2 feature list`):
* Config validation (`icinga2 daemon -C`):
* If you run multiple Icinga 2 instances, the `zones.conf` file (or `icinga2 object list --type Endpoint` and `icinga2 object list --type Zone`) from all affected nodes.

**Comment by bcogel** [comment]
I forgot..

Icingaweb2 2.5.0
Icinga2 2.8.0
RHEL 6.9

**Comment by dnsmichi** [comment]
Same happens with a local command transport like this:

```
[icinga2 cmd file]
transport = "local"
path = "/usr/local/icinga2/var/run/icinga2/cmd/icinga2.cmd" //comment
; resource = "abc"
```

2.5.x uses a more robust PHP ini parser. PHP 7 also drops the deprecated comment style with `#` and `//` and only allows for `;`.

I'll fix the docs.

**Comment by dnsmichi** [comment]
The error handling fix is to be provided with #3252.
